test/tests/functional/pbs_equiv_classes.py

```diff
@@ -1698,22 +1698,20 @@ def test_multiple_job_preemption_order(self):
 
         self.server.deljob([jid1, jid2, jid5])
 
-        workq_dict = self.server.status(QUEUE, id='workq')[0]
-        expressq_dict = self.server.status(QUEUE, id='expressq')[0]
-
-        x = 1
-        while x < 20:
-            if workq_dict.get('total_jobs') == '2' and \
-                            expressq_dict.get('total_jobs') == '1':
+        # deljob(wait=True) starts the scheduling cycle if job
+        # takes longer to be deleted.
+        # The while loop below is to check that the jobs have been deleted.
+        deleted = 0
+        for _ in range(20):
+            if self.server.expect(QUEUE, id=jid1, op=UNSET, max_attempts=None)\
+                    and self.server.expect(QUEUE, id=jid2, op=UNSET,
+                                           max_attempts=None) and\
+                    self.server.expect(QUEUE, id=jid5,
+                                       op=UNSET, max_attempts=None):
```

Comment:
I just looked through the expect() code.  If max_attempts=None, then it sets max_attempts to the default which is 60.  This won't work.  I think you need to go back to using status().